app/src/main/java/com/example/superheroes/ui/theme/Theme.kt

```diff
@@ -117,7 +115,9 @@ fun SuperheroesTheme(
     val view = LocalView.current
     if (!view.isInEditMode) {
         SideEffect {
-            setUpEdgeToEdge(view, darkTheme)
+            val window = (view.context as Activity).window
+            window.statusBarColor = colorScheme.primary.toArgb()
+            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = darkTheme
```

Comment:
```suggestion
            WindowCompat.getInsetsController(window, view).isAppearanceLightStatusBars = !darkTheme
```
Need to not operation at darkTheme to show the status information: time, battery, etc.